src/assets/scss/_logo-nav.scss

```diff
@@ -0,0 +1,57 @@
+// Logo and navigation
+
+.heading {
+  display: flex;
+  height: 45rem;
+  justify-content: space-between;
+  background-image: url(/img/background-header.webp);
+  background-size: cover;
+  padding: 6rem 8rem 6rem 8rem;
+  align-items: baseline;
+  flex-wrap: wrap;
+  background-attachment: fixed;
```

Comment:
Kiedyś (nie wiem czy nadal) to średnio wyglądało na ipadach - być może fixed będzie musiał mieć jakieś media query, ale trzeba to sprawdzić :)